keps/sig-scheduling/20180409-scheduling-framework.md

```diff
@@ -198,49 +197,51 @@ post-filter extension point.
 
 ### Scoring
 
-These plugins are used to rank nodes that have passed the filtering phase. The
-scheduler will call each scoring plugin for each node. There will be a well
-defined range of integers representing the minimum and maximum scores. After the
-[normalize scoring](#normalize-scoring) phase, the scheduler will combine node
-scores from all plugins according to the configured plugin weights.
+These plugins have two phases:
 
-### Normalize scoring
+1. The first phase is called "score" which is used to rank nodes that have passed 
+the filtering phase. The scheduler will call `Score` of each scoring plugin for
+each node.
+2. The second phase is "normalize scoring" which is used to modify scores before
+the scheduler computes a final ranking of Nodes, and each score plugin receives
+scores given by the same plugin to all nodes in "normalize scoring" phase. 
+`NormalizeScore` is called once per plugin per scheduling cycle right after 
+"score" phase.
 
-These plugins are used to modify scores before the scheduler computes a final
-ranking of Nodes. A plugin that registers for this extension point will be
-called with the [scoring](#scoring) results from the same plugin. This is called
-once per plugin per scheduling cycle.
+The output of a score plugin must be an integer in range of
+**[NodeScoreMin, NodeScoreMax]**. if not, the scheduling cycle is aborted.
```

Comment:
s/NodeScoreMin/MinNodeScore
s/NodeScoreMax/MaxNodeScore

here and all occurrences below.